src/com/goide/GoDocumentationProvider.java

```diff
@@ -85,7 +85,7 @@ else if (element instanceof GoTypeSpec) {
   }
 
   @NotNull
-  private static List<PsiComment> getCommentsInner(@Nullable PsiElement element) {
+  public static List<PsiComment> getCommentsInner(@Nullable PsiElement element) {
```

Comment:
I didn't wanted to duplicate the code in the inspection but I don't know how else to prevent getting the comment from the block and not the individual declaration with using the other method. Should I add a flag to that not to bubble up?